ocdsmerge/merge.py

```diff
@@ -83,12 +83,12 @@ def flatten(path, flattened, obj, merge_rules):
         if not iterable:
             flattened[path] = []
     for key, value in iterable:
-        # We do not flatten these keys as the child lists of
-        # these keys will not be merged, be totally replaced
-        # and versioned as a whole
         new_path = path + (key,)
-
-        if isinstance(value, (dict, list)) and 'wholeListMerge' not in merge_rules.get(remove_number_path(new_path), []):  # noqa
+        # Treat any non lists of objects (such as lists of ints, lists, strings)
+        # as whole entities so that we merge them as whole.
```

Comment:
Lists of lists are somewhat rare and unexpected, so it would be good to give the example of GeoJSON, so that the reader has an "aha, yes, list of lists make sense for GeoJSON" moment.

Reply:
Also "non lists of objects" is an odd construction to mean "lists of non-objects". Why not:

> Unless it is a list of objects, the list should be treated and merged as a whole entity. Such lists include lists of strings, lists of numbers, and lists of lists (which occur, for example, in GeoJSON fields).

Reply:
Happy for you to add a commit to this branch or against #15 in order to write comments that you think will be better understood by others. I am not sure though that this should impede this pull request.